### ISSUE TITLE
Color change for Link in Banner for A11Y

### ISSUE THREAD

**Issue by katz95** [spotify/backstage#1580]
### Currently
The link that's currently in the [dismissable banner](http://storybook.backstage.io/?path=/story/dismissablebanner--with-link) component doesn't pass A11Y standards. If we could change it to #000000, that would be great! Also the background color for the default banner should be #2E77D0. The background color for the error banner should be #E22134.

<img width="1072" alt="Screenshot 2020-07-10 at 10 50 46 AM" src="https://user-images.githubusercontent.com/61153904/87136095-444f7980-c29b-11ea-889f-292f439eab44.png">

### Ideally
<img width="643" alt="Screenshot 2020-07-10 at 10 51 54 AM" src="https://user-images.githubusercontent.com/61153904/87136164-60ebb180-c29b-11ea-9b6f-9de446a1ac9b.png">

**Comment by twishasaraiya** [comment]
Hi @katz95, I would like to help with this.

**Comment by nikek** [comment]
@katz95 I think we should collect all colors into the theme and name them semantically so that we always use named colors in components like this one and not the color hex codes directly. I'm thinking that if the regular link colors for various states doesn't work specifically in banners, we'd have to make up a new semantic color to describe colors in the context of banners, like banner.link.default, as well as for hover, active and focus states. Or can the colors be more generic like "link on colored background" to work on other areas with colored backgrounds that aren't necessarily banners?

**Comment by benjdlambert** [comment]
@twishasaraiya great I will assign you!
